packages/google_sign_in/google_sign_in/CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
-## NEXT
+## 6.1.0
 
+* Exposes the new method `canAccessScopes`.
```

Comment:
Let's mention in the CHANGELOG that it's currently only implemented for web. (Unlike method comments it's not really a problem if that becomes outdated later.)

Reply:
Good call; done!